### PR TITLE
PTL-747: FIX Chronicle Map

### PR DESCRIPTION
**Pull request by wjhendry**
This also includes a simple sentence requestReplies on Audit Tables. some other tidying while I have been there.

Thank you for contributing to the documentation.

Your Jira ticket is:
PTL-747

Have you provided changes for all the relevant versions: next, 2022.4, 2023.1, etc?
yes - where relevant

Have you checked all new or changed links?
Build successful. No links changed.

Is there anything else you would like us to know?
Olive oil production in Italy has dropped by almost 75 percent in the last ten years

For reference: 

- if you are an internal contributor:
  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 
- If you are an external contributor:
  - We have an [external contribution guide](../Type-of-contribution)

**This week's exciting excerpt from the style guide**
LISTS. Only use numbered lists when the sequence is important. Usually, that is a set of instructions.
For example:
1. Change the dictionary files.
2. Run genesisInstall.
3. Run remap.
In all other cases, use an **unnumbered** list. OK?

### CODE REVIEW

**Comment by dbarros1979** [comment]
gents (@wjhendry and @jacinpoz) we are changing this file only for 2023.1: **03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md** . Shouldn't we include it in the docs folder as well?

**Comment by wjhendry** [comment]
> gents (@wjhendry and @jacinpoz) we are changing this file only for 2023.1: **03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md** . Shouldn't we include it in the docs folder as well?

Yes, you are correct, @dbarros1979. I will change the docs as well once @jacinpoz has checked the files. It just means I don't have to implement Jose's changes twice.

File: versioned_docs/version-2023.1/03_server/03_request-server/03_advanced.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ keywords: [server, request server, advanced]
 tags:
   - server
   - request server
-  - advanced
+  - requestreply
 ---
 
-
+## Audit tables
+If you want to create a `requestReply`codeblock for an Audit table, you must specify at least one field that is an index field in the underlying table.
```
Comment (jacinpoz) [inline versioned_docs/version-2023.1/03_server/03_request-server/03_advanced.md:13]: I would add a bit more information here. "If you want to create a `requestReply` codeblock for an Audit table, and you also want to specify a specific set of `request` fields, you need to make sure the fields belong to an index in the underlying audit table.

File: versioned_docs/version-2023.1/03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md [hunk 2]
```diff
@@ -163,12 +163,31 @@ send("QUOTE_HANDLER", "QUOTE_EVENT") { quote ->
...
+or
+- if no `SENT` values are stored, the latest `COMPLETE` value 
+
+The idea is that whenever it is necessary, we can always replay events from the Streamer, starting from the oldest `SENT`(there was no response for that event and we can't guarantee it was processed successfully). If no `SENT` values are available, then it is safe to replay from the latest `COMPLETE` event. As part of this job, we also remove entries from the ChronicleMap that have been marked as `COMPLETE`.
+
+ChronicleMap instances need to be pre-allocated with a fixed set of parameters. So if the Streamer Client tries to store more than 2000 entries before the cleaning/flushing job runs (i.e. within two seconds), it exceeds the maximum capacity.
+
+For AUTH_PERMS and its internal ChronicleMap instances, the platform exposes the number of entries parameter, because we recreate the ChronicleMap file on every process restart. 
+
+However, the platform does not expose a parameter in the Streamer Client for changing the number of entries, because the cache files are persisted across Streamer Client restarts.
```
Comment (jacinpoz) [inline versioned_docs/version-2023.1/03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md:193]: the text looks good to me!

File: versioned_docs/version-2023.1/03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md [hunk 3]
```diff
@@ -163,12 +163,31 @@ send("QUOTE_HANDLER", "QUOTE_EVENT") { quote ->
         ...
     }
 }
-```
+```f
```
Comment (jacinpoz) [inline versioned_docs/version-2023.1/03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md:166]: I think this "f" shouldn't be here.

Comment (wjhendry) [reply versioned_docs/version-2023.1/03_server/10_integration/04_gateways-and-streamers/03_streamer-client.md]: i got rid of it. Thanks.

**Comment by wjhendry** [comment]
@dbarros1979 This now covers both this year's versions. It is ready for review and merge.

**Comment by wjhendry** [comment]
@dbarros1979 could you review/approve this one when you have some time today, please?

**Review by dbarros1979 — APPROVED** [review]

**Pull request merged**